jobs/create-version-branch.js

```diff
@@ -122,8 +122,9 @@ module.exports = async function (
     return
   }
 
+  let billing = null
   if (repository.private && !env.IS_ENTERPRISE) {
```

Comment:
we can send this on public repos, too. The only requirement is that they are monthly payers. Unless that makes this too complicated ;)